modules/ROOT/pages/import-api-def-dc.adoc

```diff
@@ -1,35 +1,39 @@
-= To Import an API Definition from Design Center
+= Importing an API Definition from Design Center
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 Anypoint Studio allows you to import an API Definition from Design Center and implement it either into a new Mule Project, or an existing one.
 
-== To Implement an API imported from Design Center Into a New Project
+== Implementing an API imported from Design Center Into a New Project
 
-. Click File > New > Mule Project to create a project, and set the field values in the project wizard:
+. Select *File* > *New* > *Mule Project* to create a project, and set the field values in the project wizard:
 +
-* Type a name for your project.
-* Select a Mule runtime version.
-* Check Add APIkit Components.
-* In API Definition, click the "..." button to and select Design Center.
+.. Type a name for your project.
+.. Select a Mule runtime version.
+.. Select *Specify API definition file location or URL*.
+.. To the right of the *Location* field, click the "..." button and select *Design Center*.
 . In the Browse Design Center for APIs window:
 * Select your Anypoint Platform user and business group.
 +
-Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
-. From the list of available APIs, select the one you want and click OK.
-. Click Finish.
+Logging previously into Anypoint Platform, makes your Username and Business Group details grayed out.
++
+Additionally, you can click the *Add Account* button to use a new Anypoint Platform account.
```

Comment:
Do you mean "Optionally"?

And does the button let you "use" a new account or create a new account?

Reply:
Optionally is fine. The button let you use a new account, takes the user to the Sign in to Anypoint Platform login site.

Reply:
Then, "optionally" is correct.